lms/djangoapps/courseware/tests/test_inline_analytics_integration.py

```diff
@@ -10,21 +10,21 @@
 from courseware.tests.factories import UserFactory, InstructorFactory, StaffFactory
 from xmodule.modulestore.tests.factories import CourseFactory
 from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from analyticsclient.exceptions import NotFoundError, InvalidRequestError, TimeoutError
 
 
-class InlineAnalyticsTest(ModuleStoreTestCase):
+class InlineAnalyticsTest1(ModuleStoreTestCase):
```

Comment:
Are all of these tests about answer distributions? Could we call it something like?

```python
class InlineAnalyticsAnswerDistribution(ModuleStoreTestCase):
    pass